Automattic/jetpack
# Update share limits bar design

## Proposed changes:

* Update the share limits bar design to match the Figma design - 25F3UyoNr6TUSWBplnpRyM-fi-4007%3A5874

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create/edit a post on a site without Jetpack Social plan
* Confirm that the limits bar works as expected on the sidebar of the editor
* Goto Jetpack Social admin page
* Confirm that the bar works as expected

Different scenarios

- Used: 20, Scheduled: 0

   <img width="293" alt="Screenshot 2023-12-06 at 2 33 51 PM" src="https://github.com/Automattic/jetpack/assets/18226415/128e9da3-2fed-4f04-8d26-5323a4a44a61">


- Used: 25, Scheduled: 0

    <img width="291" alt="Screenshot 2023-12-06 at 2 34 39 PM" src="https://github.com/Automattic/jetpack/assets/18226415/191e3d8f-5d10-44e5-b465-7e7c81003374">


- Used: 25, Scheduled: 3

    <img width="294" alt="Screenshot 2023-12-06 at 2 35 36 PM" src="https://github.com/Automattic/jetpack/assets/18226415/64c8864a-b49c-4f45-a808-4f7b872f5da0">


- Used: 25, Scheduled: 5

    <img width="289" alt="Screenshot 2023-12-06 at 2 36 23 PM" src="https://github.com/Automattic/jetpack/assets/18226415/796fd588-3606-46af-9f44-ed5d817577b3">


- Used: 25, Scheduled: 10

    <img width="287" alt="Screenshot 2023-12-06 at 2 37 08 PM" src="https://github.com/Automattic/jetpack/assets/18226415/6c593110-ad71-464e-be98-27e6a389e13d">


- Used: 30, Scheduled: 0

    <img width="286" alt="Screenshot 2023-12-06 at 2 37 46 PM" src="https://github.com/Automattic/jetpack/assets/18226415/a083747a-6d9d-4c6e-8471-548cb10c6bac">


- Jetpack Social admin page

    <img width="668" alt="Screenshot 2023-12-06 at 2 17 39 PM" src="https://github.com/Automattic/jetpack/assets/18226415/86a197c1-eb26-4270-b6f1-45daec0357d6">

    <img width="599" alt="Screenshot 2023-12-06 at 2 40 09 PM" src="https://github.com/Automattic/jetpack/assets/18226415/83b222ed-accb-461a-986d-69f3660983e5">

    <img width="617" alt="Screenshot 2023-12-06 at 2 41 18 PM" src="https://github.com/Automattic/jetpack/assets/18226415/7ad445da-9f4d-4785-acf5-15f64565b892">
